wagtail/wagtailforms/templates/wagtailforms/list_submissions.html

```diff
@@ -10,16 +10,22 @@
             {% for heading in data_headings %}
                 <th>{{ heading }}</th>
             {% endfor %}
+            <th>{% trans "Delete Submission" %}</th>
```

Comment:
I think it would be better to rename to "Actions" because maybe later will be added some other buttons